zipline-bytecode/src/test/kotlin/app/cash/zipline/bytecode/NoEvalTest.kt

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
This seems like a weird location for this test. Should be in `zipline/src/engineTest/` I think.